app/assets/stylesheets/pages.css.scss

```diff
@@ -319,25 +319,51 @@ $include-html-global-classes: false;
   }
 }
 
-.rackspace {
+.footer {
   background-color: $hackedu-black;
-  color: $text-on-black;
-  padding: 10px 0 10px 0;
-
-  p {
-    font-size: rem-calc(14);
-    margin: 7.5px 0 7.5px 0;
+  color: $hackedu-white;
+  padding: 30px 0;
+  font-weight: 300;
 
-    @media #{$medium-up} {
+  .nav {
+    h1 {
+      text-transform: uppercase;
```

Comment:
Properties should be ordered color, font-size, font-weight, margin-left, text-transform